src/utils/util.jl

```diff
@@ -229,6 +229,25 @@ Currently only support ReLU and Id.
 act_gradient(act::ReLU, z_hat::Vector) = z_hat .>= 0.0
 act_gradient(act::Id,   z_hat::Vector) = trues(length(z_hat))
 
+"""
+    act_gradient(act::ReLU, l::Float64, u::Float64)
+
+Returns the slope of a ReLU activation based on its lower and upper bounds
+
+Inputs:
+- `l::Float64`: lower bound
+- `u::Float64`: upper bound
+Return:
+- `slop::Float64`: 0 if u<0; 1 if l>0; u/(u-l) otherwise
+"""
+function act_gradient(act::ReLU, l::Float64, u::Float64)
```

Comment:
This signature + behavior might be misleading. Technically this isn't the activation gradient of ReLU, but of the triangle relaxation. In that sense it is different from the other `act_gradient` methods, which are not approximate. Should we rename it to `relaxed_relu_gradient(l, u)`, for example, or something else more accurate?

Reply:
yes, that is a good idea.